packages/iTwinUI-react/src/core/ColorPicker/ColorPalette.test.tsx

```diff
@@ -8,7 +8,7 @@ import { ColorPalette } from './ColorPalette';
 import { ColorPicker } from './ColorPicker';
 import { IconButton } from '../Buttons';
 import { ColorValue } from '../utils';
-import { SvgAdd } from '@itwin/itwinui-icons-react';
+import { SvgMore } from '../utils';
```

Comment:
This import can be combined with the `ColorValue` import

Reply:
Done